examples/library/genetic_algorithm/1Aug23_Standard/Ta-ACE_metrics.md

```diff
@@ -1,54 +1,54 @@
-| ('GSF_112', 'Unweighted', 'Training', 'Force')        |     1980 | 0.0429109   |   0.0731641   |      0.849975    |
-| ('GSF_112', 'weighted', 'Training', 'Energy')         |       22 | 0.00624619  |   0.00674861  |      0.846876    |
-| ('GSF_112', 'weighted', 'Training', 'Force')          |     1980 | 0.00172448  |   0.00294027  |      0.849975    |
-| ('Liquid', 'Unweighted', 'Training', 'Energy')        |        3 | 0.0370765   |   0.0371618   |      0.832859    |
-| ('Liquid', 'Unweighted', 'Training', 'Force')         |      900 | 0.223223    |   0.284315    |      0.968945    |
-| ('Liquid', 'weighted', 'Training', 'Energy')          |        3 | 0.00277174  |   0.00277811  |      0.832859    |
-| ('Liquid', 'weighted', 'Training', 'Force')           |      900 | 0.0068826   |   0.00876625  |      0.968945    |
-| ('Surface', 'Unweighted', 'Training', 'Energy')       |        7 | 0.0221882   |   0.0268803   |      0.814754    |
-| ('Surface', 'Unweighted', 'Training', 'Force')        |      708 | 0.0280469   |   0.0615215   |      0.92736     |
-| ('Surface', 'weighted', 'Training', 'Energy')         |        7 | 0.00190497  |   0.00230781  |      0.814754    |
-| ('Surface', 'weighted', 'Training', 'Force')          |      708 | 0.00182822  |   0.00401024  |      0.92736     |
-| ('Volume_A15', 'Unweighted', 'Training', 'Energy')    |       30 | 0.00998763  |   0.0125515   |      0.999983    |
-| ('Volume_A15', 'Unweighted', 'Training', 'Force')     |      720 | 1.51813e-10 |   4.84416e-10 |     -7.1093e+08  |
-| ('Volume_A15', 'weighted', 'Training', 'Energy')      |       30 | 0.00695895  |   0.00874536  |      0.999983    |
-| ('Volume_A15', 'weighted', 'Training', 'Force')       |        0 | 0           | nan           |    nan           |
-| ('Volume_BCC', 'Unweighted', 'Training', 'Energy')    |       21 | 0.0167189   |   0.0218718   |      0.999991    |
-| ('Volume_BCC', 'Unweighted', 'Training', 'Force')     |      126 | 3.67219e-14 |   7.2479e-14  | -13175.4         |
-| ('Volume_BCC', 'weighted', 'Training', 'Energy')      |       21 | 0.00911094  |   0.011919    |      0.999991    |
-| ('Volume_BCC', 'weighted', 'Training', 'Force')       |        0 | 0           | nan           |    nan           |
-| ('Volume_FCC', 'Unweighted', 'Training', 'Energy')    |       31 | 0.00609592  |   0.00755758  |      0.999997    |
-| ('Volume_FCC', 'Unweighted', 'Training', 'Force')     |      372 | 4.03487e-14 |   9.78894e-14 |     -2.84276e+06 |
-| ('Volume_FCC', 'weighted', 'Training', 'Energy')      |       31 | 0.005192    |   0.00643692  |      0.999997    |
-| ('Volume_FCC', 'weighted', 'Training', 'Force')       |        0 | 0           | nan           |    nan           |
\ No newline at end of file
+|                                                       |   ncount |             mae |            rmse |              rsq |
```

Comment:
The errors in this file are much worse than previously, was a different objective function used or something?

If so we can make a new standard.

Reply:
yes, this is expected and was due to my settings - the fits are based on a test-sized population and generation setting (only 4 and 4), so pretty terrible!

it was just to confirm that things were running

i'll re-run both ACE and SNAP with the population and generation values used in the previous examples and update the Standards directory